lib/factory_girl/strategy/stub.rb

```diff
@@ -75,6 +75,28 @@ def created_at
             end
           end
         end
+
+        responds_to_updated_at = result_instance.respond_to?(:updated_at)
+        updated_at_missing_default = responds_to_updated_at && !result_instance.updated_at
+        result_instance_missing_updated_at = !responds_to_updated_at
+
+        if updated_at_missing_default || result_instance_missing_updated_at
```

Comment:
This brings up an interesting situation - why would we add `updated_at` to an object that doesn't already have that method defined?